crates/runtime/src/logic.rs

```diff
@@ -18,6 +19,7 @@ pub type Result<T, E = errors::VMLogicError> = std::result::Result<T, E>;
 
 pub struct VMContext {
     pub input: Vec<u8>,
+    pub executor_public_key: [u8; 32],
```

Comment:
starting to think perhaps `author` might be a better name.

Reply:
We discussed this and agreed that, for now, `executor_public_key` matches the usage elsewhere, so I have left this as-is, as `author` is not used anywhere.